### PR TITLE
Upgrade gradle version

### PR DESCRIPTION
**Pull request by ningyougang**
- [x] due to openwhisk repo's gradle is upgraded to 6.9, it is better to upgrade runtime as well.

### CODE REVIEW

File: golang1.17/Dockerfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ RUN echo "deb http://deb.debian.org/debian buster-backports main contrib non-fre
      librdkafka-dev=0.11.6-1.1 &&\
     # Cleanup apt data, we do not need them later on.
     apt-get clean && rm -rf /var/lib/apt/lists/* &&\
-    go get -u github.com/go-delve/delve/cmd/dlv@v1.8.1 &&\
+    go install github.com/go-delve/delve/cmd/dlv@v1.8.1 &&\
```
Comment (ningyougang) [inline golang1.17/Dockerfile:41]: Fix issue
```
go get: installing executables with 'go get' in module mode is deprecated.
	Use 'go install pkg@version' instead.
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
# github.com/rivo/uniseg
pkg/mod/github.com/rivo/uniseg@v0.3.1/properties.go:130:6: missing function body
pkg/mod/github.com/rivo/uniseg@v0.3.1/properties.go:130:20: syntax error: unexpected [, expecting (
note: module requires Go 1.18
```

File: build.gradle [hunk 2]
```diff
@@ -18,6 +18,7 @@
 buildscript {
     repositories {
         jcenter()
+        mavenCentral()
```
Comment (ningyougang) [inline build.gradle:21]: Fix 
image
`scalafmt-core` is deprecated in jcenter, should use `mavenCentral()`
And need to keep jcenter() due to exist some lib needs this

**Comment by ningyougang** [comment]
Applied this pr's content to https://github.com/apache/openwhisk-runtime-go/pull/170/ due to this pr: https://github.com/apache/openwhisk/pull/5290

Otherwise below test case will be failed
```
runtime.actionContainers.ActionLoopBasicTests > runtime proxy should run and report an error for function not returning a json object FAILED
    org.scalatest.exceptions.TestFailedException: Some({"error":"The action did not return a dictionary."}) was not equal to Some({"error":"The action did not return a dictionary or array."})
        at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:343)
        at org.scalatest.Matchers$ShouldMethodHelper$.shouldMatcher(Matchers.scala:6723)
        at org.scalatest.Matchers$AnyShouldWrapper.should(Matchers.scala:6759)
...
```

**Comment by ningyougang** [comment]
So closed

**Comment by style95** [comment]
I think this change is one big milestone and worth having a separate commit for this.
How about merging this PR first?

**Comment by upgle** [comment]
Please reopen this PR. I also think this PR should be merged first.

**Comment by ningyougang** [comment]
@style95 @upgle i have repened, but i am not sure when the case failed whether can merge this pr. e.g. below relative test case will be failed (I tested in my local)
```
runtime.actionContainers.ActionLoopBasicTests > runtime proxy should run and report an error for function not returning a json object FAILED
    org.scalatest.exceptions.TestFailedException: Some({"error":"The action did not return a dictionary."}) was not equal to Some({"error":"The action did not return a dictionary or array."})
        at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:343)
        at org.scalatest.Matchers$ShouldMethodHelper$.shouldMatcher(Matchers.scala:6723)
        at org.scalatest.Matchers$AnyShouldWrapper.should(Matchers.scala:6759)
...
```
The reason is 
https://github.com/apache/openwhisk/pull/5290/files#diff-af225dd78c7ac7e8e1232021f772d725b544a00a87b747865af4f15eaa338768R176
image

**Comment by upgle** [comment]
@ningyougang Does this error message also occur in the current master branch?
> "The action did not return a dictionary or array"

**Comment by ningyougang** [comment]
> @ningyougang Does this error message also occur in the current master branch?
> 
> > "The action did not return a dictionary or array"

Yes

**Comment by upgle** [comment]
understand. Since it is a change of core, all sub-runtime projects will have the same issue.
Although the array return type is not supported yet, it is planned to be supported, so it seems that the message content modification can be dealt with in this PR for passing the test.

**Comment by ningyougang** [comment]
Got your

> understand. Since it is a change of core, all sub-runtime projects will have the same issue. Although the array return type is not supported yet, it is planned to be supported, so it seems that the message content modification can be dealt with in this PR for passing the test.

Got your point, fixed here: https://github.com/apache/openwhisk-runtime-go/pull/172/files#diff-c67a70867d3e68d8c9b4a4aa4ae76dc9362d2047ac0615e531d3d1bcec579892R87

**Review by upgle — APPROVED** [review]
LGTM, test cases are passed.

**Pull request merged**